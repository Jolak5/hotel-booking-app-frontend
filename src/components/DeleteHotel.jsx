import React from 'react';

const DeleteHotel = () => <div>DeleteHotel</div>;

export default DeleteHotel;
