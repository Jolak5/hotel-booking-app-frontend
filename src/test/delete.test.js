import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';

import DeleteHotel from '../components/DeleteHotel';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

describe('DeleteHotel', () => {
  const initialState = {
    home: {
      isLoading: false,
      hotels: [
        {
          id: 1,
          name: 'Hotel 1',
          image: {
            url: 'http://localhost:3000/rails/active_storage/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBCdz09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--16d773ef08a1a413ca00a961f9b8696c29001642/image1.jpg',
          },
        },
      ],
    },
  };

  const store = mockStore(initialState);

  test('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DeleteHotel />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
