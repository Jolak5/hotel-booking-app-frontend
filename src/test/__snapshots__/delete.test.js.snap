// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteHotel matches snapshot 1`] = `
<div
  className="main-del-container"
>
  <div
    className="delete-container"
  >
    <img
      alt="Hotel 1"
      className="delete-image"
      src="http://localhost:3000/rails/active_storage/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBCdz09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--16d773ef08a1a413ca00a961f9b8696c29001642/image1.jpg"
    />
    <h3
      className="delete-name"
    >
      Hotel 1
    </h3>
    <button
      className="delete-button"
      onClick={[Function]}
      type="submit"
    >
      DELETE
    </button>
  </div>
</div>
`;
